src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.css

```diff
@@ -171,6 +171,14 @@
   opacity: 1;
 }
 
+:deep(.shaka-controls-container[shown='true'] > .shaka-play-button-container > .shaka-play-button) {
+  opacity: 0.75 !important;
```

Comment:
This change should only apply when `prefers-reduced-transparency` is `no-preference` or you could add a second block to override it back to the default transparency when it is set to `reduce`.

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-transparency